docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,153 @@
+3. **Verify the peer has been join the channel successfully**. For more information, check out [Joining a channel using a snapshot](#joining-a-channel-using-a-snapshot).
+
+If an organization that is already joined to the channel wants to join a new peer using a snapshot, it might decide to skip the process of having other organizations take snapshots and evaluate them, though it is a best practice for an organization to periodically take snapshots of all its peers and compare them to ensure the no ledger forks have occurred. In that case, the organization can take a snapshot immediately and then use the snapshot to join the new peer to the channel.
+
+**Note: snapshots can also be used to verify that the public state between peers is identical, even if no new peer will use the snapshot to join the channel. This can be useful in identifying if a ledger fork has occurred.**
+
+## Taking a snapshot
+
+For the full list of snapshot-related commands, check out [`peer snapshot` commands](./commands/peersnapshot.html).
+
+Before taking a snapshot, it is a best practice to confirm the current ledger height by issuing a command similar to:
+
+```
+peer channel getinfo -c <name of channel>
+```
+
+You will see a response similar to:
+
+```
+2018-02-25 15:15:44.135 UTC [channelCmd] InitCmdFactory -> INFO 003 Endorser and orderer connections initialized
+Blockchain info: {"height":970,"currentBlockHash":"JgK9lcaPUNmFb5Mp1qe1SVMsx3o/22Ct4+n5tejcXCw=","previousBlockHash":"f8lZXoAn3gF86zrFq7L1DzW2aKuabH9Ow6SIE5Y04a4="}
+2018-02-25 15:15:44.139 UTC [main] main -> INFO 006 Exiting.....
+```
+
+In this example, the ledger height is `970`.
+
+A snapshot request can be submitted by issuing a command similar to:
+
+```
+peer snapshot submitrequest -C <name of channel> -b <ledger height where snapshot will be taken> --peerAddress <address of peer> --tlsRootCertFile <path to root certificate of the TLS CA>
+```
+
+For example:
+
+```
+peer snapshot submitrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+**If you give a ledger height of `0`, the snapshot is taken immediately. This is useful for cases when an organization is interested in backing up its own ledger data and does not intend to share the data with another organization. Do not take these "immediate" snapshots in cases when snapshots will be evaluated from multiple peers, as it increases the likelihood that the snapshots will be taken at different ledger heights.**
+
+If the request is successful, you will see a `Snapshot request submitted successfully` message.
+
+You can list the pending snapshots by issuing a command similar to:
+
+```
+peer snapshot listpending -C testchannel --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+You will see a response similar to:
+
+```
+Successfully got pending snapshot requests [970]
+```
```

Comment:
[1000]

Reply:
Sorry I meant for the submitrequest command above to be 970.

Reply:
I believe that the 1000 would have covered more regular scenario. Because typically, if you specify a particular block number, that would be well ahead of the current height. Else, by the time the snapshot request is submitted the ledger height may change and the request will not be entertained. Also, in the `cancelrequest` command below also 1000 is being used. IMO, 1000 makes better sense in this example throughout.